src/promptflow/tests/executor/unittests/processpool/test_line_execution_process_pool.py

```diff
@@ -258,3 +353,99 @@ def test_process_pool_run_with_exception(self, flow_folder, dev_connections, moc
             assert e.value.message == test_error_msg
             assert e.value.target == ErrorTarget.AZURE_RUN_STORAGE
             assert e.value.error_codes[0] == "UserError"
+
+    @pytest.mark.parametrize(
+        (
+            "flow_folder",
+            "is_set_environ_pf_worker_count",
+            "pf_worker_count",
+            "n_process"
+        ),
+        [
+            (SAMPLE_FLOW, True, "3", 3),
+            (SAMPLE_FLOW, False, None, 4)
+        ],
+    )
+    def test_process_pool_parallelism_in_fork_mode(
+            self,
+            dev_connections,
+            flow_folder,
+            is_set_environ_pf_worker_count,
+            pf_worker_count,
+            n_process):
+        p = multiprocessing.Process(target=test_fork_mode_parallelism_in_subprocess,
+                                    args=(dev_connections,
+                                          flow_folder,
+                                          is_set_environ_pf_worker_count,
+                                          pf_worker_count,
+                                          n_process))
```

Comment:
Seem that this part is not auto formatted by pre-commit?

Reply:
fixed